components/console-backend-service/internal/domain/ui/microfrontend_converter.go

```diff
@@ -0,0 +1,78 @@
+package ui
+
+import (
+	uiV1alpha1v "github.com/kyma-project/kyma/common/microfrontend-client/pkg/apis/ui/v1alpha1"
+	"github.com/kyma-project/kyma/components/console-backend-service/internal/gqlschema"
+)
+
+type microfrontendConverter struct{}
+
+func (c *microfrontendConverter) ToGQL(in *uiV1alpha1v.MicroFrontend) (*gqlschema.Microfrontend, error) {
```

Comment:
Please add changes to methods in this files described in converter for `ClusterMicrofrontend `